### DISCUSSION TITLE
getting wrong values in price filters

### DISCUSSION THREAD

**Discussion by manoharsinghrawat-cedcoss** [Shopify/storefront-api-feedback#91] [Help]
I am getting the range between 0  to 978.6 for women-tops collection whereas it has products between 1-15000 . I am using international pricing also and using India(in) in incontent parameter. Please help how can I get correct result or if I am doing anything wrong

<QueryRoot: ["collection": {
    handle = "womens-tops";
    products =     {
        filters =         (
                        {
                id = "filter.v.availability";
                label = Availability;
                type = LIST;
                values =                 (
                                        {
                        count = 89;
                        id = "filter.v.availability.1";
                        input = "{\"available\":true}";
                        label = "In stock";
                    },
                                        {
                        count = 0;
                        id = "filter.v.availability.0";
                        input = "{\"available\":false}";
                        label = "Out of stock";
                    }
                );
            },
                        {
                id = "filter.v.price";
                label = Price;
                type = "PRICE_RANGE";
                values =                 (
                                        {
                        count = 0;
                        id = "filter.v.price";
                        input = "{\"price\":{\"min\":0,\"max\":978.6}}";
                        label = Price;
                    }
                );
            }
        );
    };
}]>

**Comment by johncraigcole** [comment]
Hey @manoharsinghrawat-cedcoss when you query for the `womens-top` collection are all products in that collection available to the Online Store channel in the admin? I realize that we're chatting in the context of the Storefront API, but like needing an Online Store 2.0 theme to be installed to configure the filters, there's currently an underlying reliance on the online store channel. If products from the collection aren't active on the online store channel their prices won't be included in the filter range when querying for available filters of the collection as you are here.

**Reply by manoharsinghrawat-cedcoss** [reply]
Hello @johncraigcole thanks for replying, actually I was not using store basic currency whereas shopify filters price range is only available when customer has selected store basic currency, that was the issue, it is now fixed

**Reply by johncraigcole** [reply]
Thanks for getting back to me @manoharsinghrawat-cedcoss, glad this issue is resolved.
